index.html

```diff
@@ -36,6 +36,321 @@ <h5>LET’S CONNECT</h5>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">Back End Dev</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">2015</li>
+
+
+                  </ul>
+
+
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+
+                  <ul class="float_box1">
+
+
+                      <li class="background1"><a class="texttitle font4" href="#">html</a></li>
+                      <li class="background1"><a class="texttitle font4" href="#">css</a></li>
+                      <li class="background1"><a class="texttitle font4" href="#">javaScript</a></li>
+
+
+                  </ul>
+
+
+                  <button type="submit"><a class="font5 texttitle" href="#">See Project</a></button>
+
+
+              </div>
+
+
+          </article>
+
+
+          <article class="article3">
+
+
+              <figure>
+
+<img src="images/Snapshoot Portfolio (2).png" alt="Portfolio3">
+
+
+              </figure>
+
+
+              <div class="article3div">
+
+
+                  <h3 class="texttitle font1">Tonic</h3>
+
+
+                  <ul>
+
+
+                    <li class="texttitle font2">CANOPY</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">Back End Dev</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">2015</li>
+                  </ul>
+
+
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+
+                  <ul class="float_box1">
+
+
+                      <li class="background1"><a class="texttitle font4" href="#">html</a></li>
+                      <li class="background1"><a class="texttitle font4" href="#">css</a></li>
+                      <li class="background1"><a class="texttitle font4" href="#">javaScript</a></li>
+
+
+                  </ul>
+
+
+                  <button type="submit"><a class="font5 texttitle" href="#">See Project</a></button>
+
+
+              </div>
+
+
+          </article>
+
+
+          <article class="article4">
+
+
+              <figure>
+
+<img src="images/Snapshoot Portfolio (3).png" alt="Portfolio4">
+
+
+              </figure>
+
+
+              <div class="article4div">
+
+
+                  <h3 class="font1">Multi-Post Stories</h3>
+
+
+                  <ul>
+
+
+                    <li class="texttitle font2">CANOPY</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">Back End Dev</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">2015</li>
```

Comment:
- Kindly, be sure that the bullet icons are visible 👍👌 and it is best to use style instead of image to display bullets👌👍